.github/workflows/main.yml

```diff
@@ -34,11 +34,35 @@ jobs:
         run: |
           wget --no-verbose http://www.crc.nd.edu/~kmohrman/files/root_files/for_ci/ttHJet_UL17_R1B14_NAOD-00000_10194.root
 
-      - name: Run processors over test files with futures executor
+      - name: Test unit
         run: |
           conda install -y -n coffea-env -c conda-forge pytest pytest-cov root_base
           mkdir histos
-          conda run -n coffea-env pytest --cov=./ --cov-report=xml -rP
```

Comment:
This change removes our code coverage check. If you add back the `--cov=./ --cov-report=xml -rP` part to the _first_ `pytest` line, and `--cov=./ --cov-report=xml -rP --cov-append` should hopefully work.